README.md

```diff
@@ -22,6 +22,22 @@ Available in [COPR](https://copr.fedorainfracloud.org/coprs/atim/diskonaut/):
 sudo dnf copr enable atim/diskonaut -y
 sudo dnf install diskonaut
 ```
+
+### Arch Linux
+
+Available in the [AUR](https://aur.archlinux.org/diskonaut):
```

Comment:
Hey, I'm getting a 404 on this... is it maybe https://aur.archlinux.org/packages/diskonaut ?

Reply:
Yeah, sorry about that. Just pushed a fix